content/configuring/anomaly_scoring.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Anomaly Scoring"
+chapter: false
+weight: 10
+---
+
+> The Core Rule Set 3 is designed as an anomaly scoring rule set. This page explains what anomaly scoring is and how to use it.
+
+## Overview of Anomaly Scoring
+
+Anomaly scoring, also known as "collaborative detection", is a scoring mechanism used in the Core Rule Set. It assigns a numeric score to HTTP transactions (requests and responses), representing how 'anomalous' they appear to be. Anomaly scores can then be used to make blocking decisions. The default CRS blocking policy, for example, is to block any transaction that meets or exceeds a defined anomaly score threshold.
+
+## How Anomaly Scoring Mode Works
+
+Anomaly scoring mode combines the concepts of *collaborative detection* and *delayed blocking*. The key idea to understand is that **the inspection/detection rule logic is decoupled from the blocking functionality**.
+
+Individual rules designed to detect specific types of attacks and malicious behaviour are executed. If a rule matches, no immediate disruptive action is taken (e.g. the transaction is not blocked). Instead, the matched rule contributes to a transactional *anomaly score*, which acts as a running total. The rules just handle detection, adding to the anomaly score if they match. In addition, an individual matched rule will typically log a record of the match for later reference, including the ID of the matched rule, the data that caused the match, and the URI that was being requested.
+
+Once all of the rules that inspect *request* data have been executed, *blocking evaluation* takes place. If the anomaly score is greater than or equal to the inbound anomaly score threshold then the transaction is *denied*. Transactions that are not denied continue on their journey.
+
+![Diagram showing an example where the inbound anomaly score threshold is set to 5. A first example request accumulates an anomaly score of 7 and is denied at the blocking evaluation step. A second example request accumulates no anomaly score and is allowed to pass at the blocking evaluation step.](as_inbound_no_fonts.svg?height=36em)
+
+Continuing on, once all of the rules that inspect *response* data have been executed, a second round of blocking evaluation takes place. If the *outbound* anomaly score is greater than or equal to the outbound anomaly score threshold then the transaction is *denied*.
+
+{{% notice info %}}
+Having separate inbound and outbound anomaly scores and thresholds allows for request data and response data to be inspected and scored independently.
+{{% /notice %}}
+
+### Summary of Anomaly Scoring Mode
+
+To summarise, anomaly scoring mode in the CRS works like so:
```

Comment:
How painful would it be for you to write the American form? I'm happy with the English orthography form. But updates are likely to be in the American form and then we end up with a mix.

Reply:
If this is the language standard that the project is going to use that's fine with me. I **will** make mistakes, though, so I'll see if I can add a British English grep check or something as a GitHub Action haha.

I'll make a note to document that American English should be used, for the attention of future contributors/joiners.